### ISSUE TITLE
Improve Litmus documentatiom for Resilience probes

### ISSUE THREAD

**Issue by Saranya-jena** [litmuschaos/litmus#4378]
Here are few points where we can improve upon:
- In resilience probe section we can add few points to mention that probe name is unique and can't be reused as it is an identifier. Deleting the probe will disable it not force delete it. - This can be updated in the documentation.
- While creating an experiment , in user guide section we can add the step for probe addition and also mention that it is now mandatory.
- In experiment creation we need a probeRef in annotations to add res probes (if constructing manually or uploading YAML). This can also be mentioned.

**Comment by anshikavashistha** [comment]
@Saranya-jena May I work on this issue?

**Comment by Saranya-jena** [comment]
Hi @anshikavashistha sure, you can take it up. You will need to raise PR here: https://github.com/litmuschaos/litmus-docs

**Comment by anshikavashistha** [comment]
Surely 
Thank you

**Comment by anshikavashistha** [comment]
https://github.com/amityt/litmus-docs-beta/blob/4efd5c3bd38339d4a6306b389cfcfc1871570ccd/website/docs/concepts/probes.md
I think all the improvements should be updated there

**Comment by neelanjan00** [comment]
> https://github.com/amityt/litmus-docs-beta/blob/4efd5c3bd38339d4a6306b389cfcfc1871570ccd/website/docs/concepts/probes.md I think all the improvements should be updated there

No, please add it to the Resilience Probe section: https://docs.litmuschaos.io/docs/user-guides/create-resilience-probe

**Comment by anshikavashistha** [comment]
Sure @neelanjan00 I will add it to the Resilience Probe section
Thankyou
